control_panel_api/tests/test_aws.py

```diff
@@ -4,7 +4,7 @@
 from control_panel_api import aws
 
 
-@patch('boto3.client')
+@patch('control_panel_api.aws.aws_api_client')
```

Comment:
Is `@patch()` basically stubbing the argument to return a `MagicMock` instance behind the scenes?

As said in other comment, ideally if we can do that transparently when running tests, we may well remove this line (or forget by accident...) and any `boto3.client()` call would return the mock instead of a genuine client.

Reply:
This is what it is doing now. The patch was just there as that was how you inject the ability to assert against the mock, but as Andy pointed out this is now redundant as the client is a mock by default so I can just import it (i'm updating code now).